js/main.js

```diff
@@ -5260,17 +5260,29 @@ function eventDescription(tracks) {
             var talk_id = this.id.substring(6,this.id.length);
             var talk_title = tracks[String(talk_id)]["title"];
             var talk_description = tracks[String(talk_id)]["description"];
+            let socials, icons = [];
             if(tracks[String(talk_id)]["speaker"]){
                 var talk_speaker_name = tracks[String(talk_id)]["speaker"]["name"];
                 var talk_speaker_info = tracks[String(talk_id)]["speaker"]["info"];
                 var talk_speaker_image = tracks[String(talk_id)]["speaker"]["photo"];
+
+                // If speaker has any socials
+                socials = tracks[String(talk_id)]["speaker"]["social"][0] ?? []
+                for (const [platform, url] of Object.entries(socials)) {
+                    console.log(`${platform}: ${url}`);
+                    if (url.toString().trim() == "") continue;
```

Comment:
```suggestion
                    if (url.toString().trim() == "") continue;
```
Would have preferred normalizing the source data itself but this is fine for now.